.docker/entrypoint.sh

```diff
@@ -5,3 +5,7 @@ usermod --non-unique --uid "${HOST_UID}" www-data
 groupmod --non-unique --gid "${HOST_GID}" www-data
 
 php /var/www/html/vendor/bin/jigsaw serve --host=0.0.0.0
+
+if [ ! -d "vendor" ] then
+  composer install
+fi
```

Comment:
Acredito que só vamos conseguir rodar o Jigsaw depois de instalar as dependências. 

```suggestion
if [ ! -d "vendor" ] then
  composer install
fi

php /var/www/html/vendor/bin/jigsaw serve --host=0.0.0.0
```